doc/jsdoc.tsx

```diff
@@ -17,13 +17,13 @@ export function JsDoc(
   return <Markdown context={context}>{jsDoc.doc}</Markdown>;
 }
 
-export function JsDocModule(
-  { children, context }: {
-    children: Child<DocNodeTupleArray<DocNodeModuleDoc>>;
-    context: Context;
-  },
-) {
-  const moduleDoc = take(children, true, true);
+export function getJsDocModule(
```

Comment:
nit: How about moving this to util.ts?

Reply:
yea was thinking about that; will do it. will also move the JsDoc function to the markdown file; shouldnt need to have its own file for like 10 lines of code

Reply:
@kt3k I decided to remove that function completely as its used in a single place. while i was at it i moved the jsdoc component to the markdown file.